content/v1.10-docs/installation/helm.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Helm
+description: 'cert-manager installation: Using Helm'
+---
+
+## Installing with Helm
+
+cert-manager provides Helm charts as a first-class method of installation on both Kubernetes and OpenShift.
+
+Be sure never to embed cert-manager as a sub-chart of other Helm charts; cert-manager manages
+non-namespaced resources in your cluster and care must be taken to ensure that it is installed exactly once.
+
+### Prerequisites
+
+- [Install Helm version 3 or later](https://helm.sh/docs/intro/install/).
+- Install a [supported version of Kubernetes or OpenShift](./supported-releases.md).
+- Read [Compatibility with Kubernetes Platform Providers](./compatibility.md) if you are using Kubernetes on a cloud platform.
+
+### Steps
+
+
+#### 1. Add the Helm repository
+
+This repository is the only supported source of cert-manager charts. There are some other mirrors and copies across the internet, but those are entirely unofficial and could present a security risk.
+
+Notably, the "Helm stable repository" version of cert-manager is deprecated and should not be used.
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+```
+
+#### 2. Update your local Helm chart repository cache:
+
+```bash
+helm repo update
+```
+
+#### 3. Install `CustomResourceDefinitions`
+
+cert-manager requires a number of CRD resources, which can  be installed manually using `kubectl`,
+or using the `installCRDs` option when installing the Helm chart.
+
+##### Option 1: installing CRDs with `kubectl`
+
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.crds.yaml
+```
+
+##### Option 2: install CRDs as part of the Helm release
+
+To automatically install and manage the CRDs as part of your Helm release, you
+must add the `--set installCRDs=true` flag to your Helm installation command.
+
+Uncomment the relevant line in the next steps to enable this.
+
+Note that if you're using a `helm` version based on Kubernetes `v1.18` or below (Helm `v3.2`), `installCRDs` will not work with cert-manager `v0.16`. See the [v0.16 upgrade notes](./upgrading/upgrading-0.15-0.16.md#helm) for more details.
+
+#### 4. Install cert-manager
+
+To install the cert-manager Helm chart, use the [Helm install command](https://helm.sh/docs/helm/helm_install/) as described below.
+
+```bash
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.9.1 \
+  # --set installCRDs=true
```

Comment:
suggestion: (another thing which should be automated :sob: ) there are some hardcoded versions in installation docs which might need bumping to reference the latest version, in `/v1.10-docs` and also in `/docs` and `/next-docs`.